docs/images.coffee.md

```diff
@@ -21,7 +21,7 @@ Here is an example showing some of these options.
        .text('Proprotional to width', 0, 0)
      
     # Fit the image within the dimensions
-    doc.image('images/test.jpeg', 320, 15, fit: [100, 100])
+    doc.image('data:image/png;base64,iVBORw0K...', 320, 15, fit: [100, 100])
```

Comment:
Since these examples are actually run and the output is displayed inline (below), this code will not work truncated. I think it would look pretty bad to display the whole thing, so maybe just leave the code as it was. base64 support is mentioned above, which is probably enough.